format.js

```diff
@@ -32,24 +34,36 @@ function makeEncryptionFormat() {
     )
     Keyring(keyringPath, (err, api) => {
       if (err) return cb(err)
-      _keyring = api
-      _keyring.dm.addFromSSBKeys(config.keys)
-      _keyringReady.setReady()
+      keyring = api
+      mainKeys = config.keys
+      keyringReady.setReady()
       cb()
     })
   }
 
   function teardown(cb) {
-    _keyringReady.onReady(() => {
-      _keyring.close(cb)
+    keyringReady.onReady(() => {
+      keyring.close(cb)
     })
   }
 
-  function _isGroup(recp) {
-    return _keyring.group.has(recp)
+  function disableLegacyMode() {
+    legacyMode = false
   }
 
-  function _isFeed(recp) {
+  function isPrimordialGroup(recp) {
```

Comment:
Maybe isGroupRecp instead?

Reply:
Gotta find a way of naming the difference between groupId (cloaked id) as a recp, and bare object (`{key,scheme}`) as a recp.

Reply:
I named it now `isRawGroupKey`